scss/_buttons.scss

```diff
@@ -25,6 +25,11 @@
   .octicon {
     vertical-align: text-top;
   }
+  
+  // Darken for just a tad more contrast against the button background
+  .counter {
+    background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.